doc/source/index.rst

```diff
@@ -1,67 +1,67 @@
    :target: https://github.com/IAMconsortium/nomenclature/blob/main/LICENSE
 
-.. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
-   :target: https://github.com/psf/black
+.. |doi| image:: https://zenodo.org/badge/375724610.svg
+   :target: https://zenodo.org/badge/latestdoi/375724610
 
 .. |python| image:: https://img.shields.io/badge/python-_3.8_|_3.9_|_3.10-blue?logo=python&logoColor=white
    :target: https://github.com/IAMconsortium/nomenclature
 
+.. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
+   :target: https://github.com/psf/black
+
 .. |pytest| image:: https://github.com/IAMconsortium/nomenclature/actions/workflows/pytest.yml/badge.svg
    :target: https://github.com/IAMconsortium/nomenclature/actions/workflows/pytest.yml
 
 .. |rtd| image:: https://readthedocs.org/projects/nomenclature-iamc/badge/?version=latest
    :target: https://nomenclature-iamc.readthedocs.io
 
 Overview
-========
-
-The nomenclature package facilitates working with "codelists" that follow the format
-developed by the `Integrated Assessment Modeling Consortium (IAMC)
-<https://www.iamconsortium.org>`_. Codelists are yaml file based lists of allowed values
-(or codes) for dimensions of IAMC-style data, for example *regions* and *variables*.
-Using these codelists, nomenclature performs data validation to check if a provided data
-set conforms to the values in the code lists. 
-
-Additionally, it can execute "region processing", which consists of renaming of "native
-regions" and/or aggregation to "common regions" used in a project.
+--------
 
-Those two tasks are carried out by two classes:
+The **nomenclature** package facilitates validation and processing of scenario data
+for model comparison projects and scenario analysis. It allows to manage
+project templates and "codelists" that follow the format developed by the
+`Integrated Assessment Modeling Consortium (IAMC) <https://www.iamconsortium.org>`_.
 
-#. The :class:`DataStructureDefinition` class handles the validation of scenario data.
-   It contains data templates for *variables* (including units) and *regions* to be used
-   in a model comparison or scenario exercise following the IAMC data format.
+A "codelist" is a list allowed values (or "codes") for dimensions of IAMC-format data,
+typically *regions* and *variables*. Each code can have additional attributes:
+for example, a "variable" (string) usually has a definition and an expected unit.
+Read the `SDMX Guidelines <https://sdmx.org/?page_id=4345>`_ for more information on
+the concept of codelists.
 
-#. The :class:`RegionProcessor` class carries out renaming and aggregation based on
-   information given in yaml model mapping files.
+The **nomenclature** package supports three main use cases:
 
-Instructions on how to install nomenclature can be found in the "Installation" section.
+- Management of codelists, definitions and mappings for model comparison projects
```

Comment:
I guess this is mostly copied from README.md so the same comment as above.
What do you mean by definitions as opposed to codelists? Did we explain what a "mapping" is in this context?
I'm not sure about the "management" aspect of nomenclature.

Reply:
Well, the Readme is a shortened copy of the index page because they both serve the purpose of a high-level overview of the purpose... Removed the word "definitions" from this list.

Reply:
Ah yes, sorry for the confusion I wasn't criticizing that the fact that the REAMDE and the index are similar. I meant it only in reference to my "definitions" comment.